larray/core/axis.py

```diff
@@ -2097,39 +2099,99 @@ def combine_axes(self, axes=None, sep='_', wildcard=False, front_if_spread=False
         -------
         AxisCollection
             New AxisCollection with combined axes.
+            
+        Examples
+        --------
+        >>> axes = AxisCollection('a=a0..a2;b=b0..b2')
```

Comment:
use a0,a1 instead to make examples shorter and prevent people from thinking axes must have the same length to be combined?